xapi_db_load/backends/clickhouse_lake.py

```diff
@@ -42,23 +43,23 @@ def drop_tables(self):
         print("Tables dropped")
 
     def create_tables(self):
-        return
         sql = f"""
             CREATE TABLE IF NOT EXISTS {self.event_table_name} (
             event_id UUID NOT NULL,
             verb_id String NOT NULL,
             actor_id UUID NOT NULL,
             org String NOT NULL,
-            -- course_id String NOT NULL,
-            -- problem_id String NULL,
-            -- video_id String NULL,
-            -- nav_starting_point String NULL,
-            -- nav_ending_point String NULL,
+            course_id String NOT NULL,
```

Comment:
These changes are all fine, this is mostly a note for myself to come back and change the rest of this create statement to the one that was performing better in the Ralph tests:

```
            CREATE TABLE IF NOT EXISTS {self.event_table_name} (
            event_id UUID NOT NULL,
            verb_id String NOT NULL,
            actor_id UUID NOT NULL,
            org String NOT NULL,
            course_id String NOT NULL,
            emission_time timestamp NOT NULL,
            event JSON NOT NULL
            )
            ENGINE MergeTree ORDER BY (org, course_id, verb_id, actor_id, emission_time,
             event_id)
            PRIMARY KEY (org, course_id, verb_id, actor_id, emission_time, event_id)
```

Reply:
Ah cool -- I've left the extra fields alone (since the data loading requires them), but c106f35b06b414fea1b0450bcb4b9629bcfa6e63 adds those additional fields to `ORDER BY` and `PRIMARY KEY`.

Reply:
Ok, thanks. I'll normalize those between the clickhouse and Ralph backends in a different PR. That table structure is going to all need to be updated after some discussions I just had with FUN, but I haven't decided on exactly how yet.